ydb/docs/en/core/reference/ydb-sdk/health-check-api.md

```diff
@@ -38,37 +49,98 @@ message IssueLog {
+| `PDisk is not available` | A physical disk is not available. |
+| **STORAGE_NODE** ||
+| `Storage node is not available` | A node with disks is not available. |
+| **COMPUTE** ||
+| `There are no compute nodes` | The database has no nodes to start the tablets. </br>Unable to determine `COMPUTE_NODE` issues below. |
+| `Compute has issues with system tablets` | These issues depend solely on the underlying `SYSTEM_TABLET` layer. |
+| `Some nodes are restarting too often` | These issues depend solely on the underlying `NODE_UPTIME` layer. |
+| `Compute is overloaded` | These issues depend solely on the underlying `COMPUTE_POOL` layer. |
+| `Compute quota usage` | These issues depend solely on the underlying `COMPUTE_QUOTA` layer. |
+| `Compute has issues with tablets`| These issues depend solely on the underlying `TABLET` layer. |
+| **COMPUTE_QUOTA** ||
+| `Paths quota usage is over than 90%/99%/Paths quota exhausted` </br>`Shards quota usage is over than 90%/99%/Shards quota exhausted` |Quotas exhausted|
+| **COMPUTE_NODE** | *There is no specific issues on this layer.* |
+| **SYSTEM_TABLET** ||
+| `System tablet is unresponsive / response time over 1000ms/5000ms`|  The system tablet is not responding or it takes too long to respond. |
+| **TABLET** ||
+| `Tablets are restarting too often` | Tablets are restarting too often. |
+| `Tablets/Followers are dead` | Tablets are not running (probably cannot be started). |
+| **LOAD_AVERAGE** ||
+| `LoadAverage above 100%` | A physical host is overloaded. </br> The `Healthcheck` tool monitors system load by evaluating the current workload in terms of running and waiting processes (load) and comparing it to the total number of logical cores on the host (cores). For example, if a system has 8 logical cores and the current load value is 16, the load is considered to be 200%. </br> `Healthcheck` only checks if the load exceeds the number of cores (load > cores) and reports based on this condition. This indicates that the system is working at or beyond its capacity, potentially due to a high number of processes waiting for I/O operations. </br></br> Load Information: </br> Source: </br>`/proc/loadavg` </br> Logical Cores Information </br></br>The number of logical cores: </br>Primary Source: </br>`/sys/fs/cgroup/cpu.max` </br></br>Fallback Source: </br>`/sys/fs/cgroup/cpu/cpu.cfs_quota_us` </br> `/sys/fs/cgroup/cpu/cpu.cfs_period_us` </br>The number of cores is calculated by dividing the quota by the period (quota / period)
+| **COMPUTE_POOL** ||
+| `Pool usage is over than 90/95/99%` | One of the pools' CPUs is overloaded. |
+| **NODE_UPTIME** ||
+| `Node is restarting too often/The number of node restarts has increased` | The number of node restarts has exceeded the threshold. |
+| **NODES_SYNC** ||
+| `The nodes have a time difference of ... ms` | Time drift on nodes might lead to potential issues with coordinating distributed transactions. |
```

Comment:
It needs some details about what level of time difference is acceptable, maybe via a link to other content.